### PR TITLE
Update PostProcessor.py

### PR DESCRIPTION
**Pull request by PipKat**
Change class and function references to single backtick per Numpy and minor edits.

### CODE REVIEW

File: pyaedt/modules/PostProcessor.py [hunk 1]
```diff
@@ -744,7 +747,7 @@ def change_field_property(self,plotname, propertyname, propertyval):
         propertyname : str
             Name of the property to edit.   
         propertyval :
-            New value for the property.
+           Value for the property.
```
Comment (akaszynski) [inline pyaedt/modules/PostProcessor.py:750]:
```suggestion
            Value for the property.
```

File: pyaedt/modules/PostProcessor.py [hunk 2]
```diff
@@ -1409,20 +1410,20 @@ def get_report_data(self, expression="dB(S(1,1))", setup_sweep_name='', domain='
...
+            in which case the nominal sweep is used.
         domain : str or list, optional
             Context type. The options are ``"Sweep"`` or ``"Time"``. The default is ``"Sweep".``
         families_dict : dict, optional
             Dictionary of all families including the primary sweep. The default is ``{"Freq": ["All"]}``.
         report_input_type :  str
-             Type of data.
-
+             Type of input data for the report.
+
```
Comment (akaszynski) [inline pyaedt/modules/PostProcessor.py:1421]:
```suggestion

```

File: pyaedt/modules/PostProcessor.py [hunk 3]
```diff
@@ -1409,20 +1410,20 @@ def get_report_data(self, expression="dB(S(1,1))", setup_sweep_name='', domain='
...
-
+             Type of input data for the report.
+        
         Returns
         -------
         SolutionData
-            SolutionData object if successful
-
+            SolutionData object if successful.
+
```
Comment (akaszynski) [inline pyaedt/modules/PostProcessor.py:1426]:
```suggestion

```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

File: pyaedt/modules/PostProcessor.py [hunk 4]
```diff
@@ -1409,20 +1410,20 @@ def get_report_data(self, expression="dB(S(1,1))", setup_sweep_name='', domain='
         expression : str or list
             One or more formulas to add to the report. The default is ``"dB(S(1,1))"``.
         setup_sweep_name : str, optional
-            Name of setup to use to compute the report. The default is ``""``, which means that the
-            nominal sweep is to be applied.
+            Name of setup to use to compute the report. The default is ``""``,
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:1413]:
```suggestion
            Name of the setup to use to compute the report. The default is ``""``, 
```

**Pull request merged**